onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -3572,17 +3572,87 @@ def aten_max_pool1d_with_indices(
     raise NotImplementedError()
 
 
+@torch_op("aten::max_pool2d", trace_only=True)
 def aten_max_pool2d(
     self: TensorType,
     kernel_size: Sequence[int],
-    stride: Optional[Sequence[int]] = None,
-    padding: Sequence[int] = (0, 0),
+    stride: Sequence[int] = None,
+    padding: Sequence[int] = 0,
```

Comment:
```suggestion
    padding: Sequence[int] = (0, 0),
```

ok to keep (0, 0)?

Reply:
No matter what it is, but should be align with padding's format.